src/main/java/org/example/homework_nr_8/Animal.java

```diff
@@ -0,0 +1,19 @@
+package org.example.homework_nr_8;
+
+public interface Animal {
+
+    default double foodCosts(double income){
+        return income * 0.8;
```

Comment:
That's a correct implementation, but now you have to create a variable inside the interface and use it here.

ex.
`float FOOD_COST_COEFICIENT = 0.8`

and then:

`return income * FOOD_COST_COEFICIENT`

This way is recommended because if you would want to use this value somewhere else you have a final variable here.